unskript-ctl/templates/template_script.j2

```diff
@@ -14,7 +14,11 @@ def _run_function(fn):
         last_cell()
         output = output_buffer.getvalue()
     else:
-        eval(fn)
+        try:
+            eval(fn)
+        except:
+            # If one of the action fails, proceed further
```

Comment:
How is the error captured for a lego which has failed?

Reply:
Dumping the Error / Exception on to stdout which is captured and displayed.